lib/dRep/dRepImpl.py

```diff
@@ -1,24 +1,42 @@
 # -*- coding: utf-8 -*-
 #BEGIN_HEADER
+####################################################################################################
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+#---------------------------------------------------------------------------------------------------
+
+
 import logging
 import os
+import sys
 import shutil
 import subprocess
 import pprint
 import uuid
-
+import re
+import functools
+import pickle
+import warnings
 
 from installed_clients.KBaseReportClient import KBaseReport
 from installed_clients.DataFileUtilClient import DataFileUtil
 from installed_clients.WorkspaceClient import Workspace
 from installed_clients.MetagenomeUtilsClient import MetagenomeUtils
 
-
-
-from .util import *
+from .util import PrintUtil, KBaseObjUtil, OutputUtil
 from .util.PrintUtil import *
```

Comment:
I would avoid any star imports. They put a whole bunch of names in your namespace implicitly and it can be really hard to trace where anything comes from. Make all imports explicit
See pep8 https://pep8.org/#imports

Reply:
i did feel at some point stars were sloppy